app/services/displayable_pii_formatter.rb

```diff
@@ -0,0 +1,81 @@
+class DisplayablePiiFormatter
+  attr_reader :current_user
+  attr_reader :pii
+
+  def initialize(current_user:, pii:)
+    @current_user = current_user
+    @pii = pii
+  end
+
+  def format
+    formatted_pii = {
+      email: email,
+      all_emails: all_emails,
+      verified_at: verified_at,
+      x509_subject: x509_subject,
+      x509_issuer: x509_issuer,
+    }
+    formatted_pii.merge!(formatted_ial2_pii) if pii.any?
+    formatted_pii
+  end
```

Comment:
A few ideas:
1. let's make this return a Struct or something so we can minimize digging around in hashes?
2. what we just called the property methods on this directly? ex `DisplayablePiiFormatter.new(pii).first_name`

Reply:
That's a good idea, I'll get started on that